third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -756,6 +756,12 @@ pub enum StatusCode {
     RESERVED_RUNTIME_ERROR_4 = 4036,
     RESERVED_RUNTIME_ERROR_5 = 4037,
 
+    // Error happens due to error in communicating to the remote executor service
+    REMOTE_EXECUTION_SERVER_READ_ERROR = 5000,
+    REMOTE_EXECUTION_SERVER_WRITE_ERROR = 5001,
+    REMOTE_REQUEST_SERIALIZATION_ERROR = 5002,
+    REMOTE_RESPONSE_DESERIALIZATION_ERROR = 5003,
+
```

Comment:
Not sure we want to introduce these. Especially not in the 5xxx range, see the error code categorization code below. 
These errors are supposed to be part of the VM transaction output, I assume it's irrelevant to your change anyway?

Reply:
You are right, these doesn't belong here. Let me clean them up.